frontend/src/components/Card.tsx

```diff
@@ -60,11 +60,10 @@ const Card = ({
             </span>
           )}
           <a href={url} target="_blank" rel="noopener noreferrer">
-            <h1 className="text-2xl font-semibold dark:text-sky-600">{title}</h1>
+            <h1 className="text-lg font-semibold sm:text-2xl dark:text-sky-600">{title}</h1>
           </a>
         </div>
-
-        <div className="flex min-w-[30%] flex-wrap items-center justify-end">
+        <div className="flex w-full flex-wrap items-center justify-end gap-2 sm:w-auto">
           {icons &&
             Object.keys(Icons).map((key, index) =>
```

Comment:
Also, these are now don't look good on the mobile view. I'm thinking for a mobile device view we should switch to something like a list items instead. 
We can address that in a separate task though! 👍🏼